201-vm-customimage-new-storageaccount/azuredeploy.json

```diff
@@ -0,0 +1,307 @@
+  "parameters": {
+    "TransferVmName": {
+      "type": "string",
+      "defaultValue": "TransferVM",
+      "minLength": 3,
+      "maxLength": 15,
+      "metadata": {
+        "description": "Name of the Windows VM that will perform the copy of the VHD from a source storage account to the new storage account created in the new deployement, this is known as transfer vm."
+      }
+    },
+    "NewVmName": {
+      "type": "string",
+      "defaultValue": "NewVM",
+      "minLength": 3,
+      "maxLength": 15,
+      "metadata": {
+        "description": "Name of the new VM deployed from the custom image."
+      }
+    },
+    "CustomImageName": {
+      "type": "string",
+      "defaultValue": "images/Win10MasterImage-osDisk.72451a98-4c26-4375-90c5-0a940dd56bab.vhd",
+      "metadata": {
+        "description": "Name of the VHD to be used as source syspreped/generalized image to deploy the VM. E.g. mybaseimage.vhd."
+      }
+    },
+    "AdminUserName": {
+      "type": "string",
+      "defaultValue": "vadmin",
+      "minLength": 1,
+      "metadata": {
+        "description": "Name of the local administrator account, this cannot be Admin or Administrator or root if Linux VM."
+      }
+    },
+    "AdminPassword": {
+      "type": "securestring",
+      "minLength": 8,
+      "metadata": {
+        "description": "Local administrator password, complex password is required, do not use any variation of the password word because it will be rejected."
+      }
+    },
+    "sourceStorageAccountKey": {
+      "type": "string",
+      "metadata": {
+        "description": "Key of the storage account that contains the custom image."
+      }
+    },
+    "sourceImages": {
+      "type": "string",
+      "defaultValue": "https://jumpvmrg7942.blob.core.windows.net/vhds/CustomMasterImage-osDisk.72451a98-4c26-4375-90c5-0a940dd56bab.vhd,https://jumpvmrg7942.blob.core.windows.net/vhds/CustomDisk01.vhd",
+      "metadata": {
+        "description": "Full URIs for one or more custom images (VHDs) that should be copied to the deployment storage account to spin up new VMs from them. URLs must be comma separated."
+      }
+    }
+  },
+  "variables": {
+    "vmCount": 2,
+    "vmNames": [
+      "[parameters('transferVmName')]",
+      "[parameters('newVmName')]"
+    ],
+    "nicNames": [
+      "[concat(parameters('transferVmName'),'Nic')]",
+      "[concat(parameters('newVmName'),'Nic')]"
+    ],
+    "storageAccountType": "Standard_LRS",
+    "storageAccountName": "[concat('storageacct',uniquestring(resourceGroup().id))]",
+    "virtualNetworkName": "vNet",
+    "vnetPrefix": "10.0.0.0/16",
+    "vnetSubnetName1": "LabSubnet",
+    "vnetSubnetPrefix1": "10.0.0.0/24",
+    "VnetID": "[resourceId('Microsoft.Network/virtualNetworks', variables('VirtualNetworkName'))]",
+    "SubnetRef": "[concat(variables('VnetID'), '/subnets/', variables('vnetSubnetName1'))]",
+    "projectName": "201-vm-customimage-new-storageaccount",
```

Comment:
Template should be called 201-vm-custom-image-new-storage-account for better readability